src/utils/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright © 2017-2020 Canonical Ltd.
+# Copyright © 2017-2021 Canonical Ltd.
```

Comment:
This isn't wrong, since once you change it, it's been changed :) But not necessary either.

Reply:
That said, I don't mind if you prefer to leave it like that. I wouldn't mind even doing a commit changing it in all files each year. It would add to each file history, but that wouldn't be a problem IMO. Anyway, as you prefer here.

Reply:
Yeah it was a leftover, fixed :)